.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
Nope, Amazon isn't doing anything with it now. Everything is routed through API Gateway and it's treated the same as if it wasn't a real lambda as far as I can tell.